src/workers/cdc.ts

```diff
@@ -312,6 +325,25 @@ const onReputationEventChange = async (
   }
 };
 
+const onSubmissionChange = async (
+  con: Connection,
+  logger: FastifyLoggerInstance,
+  data: ChangeMessage<Submission>,
+) => {
+  const entity = data.payload.after;
+  if (data.payload.op === 'c') {
+    await notifySubmissionCreated(logger, {
+      sourceId: 'TBD',
+      url: entity.url,
+      submissionId: entity.id,
+    });
+  } else if (data.payload.op === 'u') {
+    if (entity.status !== SubmissionStatus.Accepted) {
```

Comment:
Why?

Reply:
We used to send the mail for when the post is created. Though after the review, it seems to be better off when the submission changed to accepted, this condition should now be omitted.